setup.py

```diff
@@ -27,7 +28,7 @@
 
 setup(
     name='salesforce-oauth-request',
-    version='1.0.6',
+    version='1.1.0',
```

Comment:
normally I increment version in separate commit so it's easy to track down. probably not worth doing until you fork to `-yplan`